### PR TITLE
Should fix #61

### LINKED ISSUE #61: missing libgcc
In our minimal Docker builds we see this error if we are using matplotlib.

```
[Sep 26 22:27:46] STEP Run image [quay.io/mulled/multiqc:0.8dev0--py35_0] with command [[multiqc --version]]
[Sep 26 22:27:46] DEBU Creating container [step-66ab04b33d]
[Sep 26 22:27:49] DEBU Created container [e07531f9e567 step-66ab04b33d], starting it
[Sep 26 22:27:57] DEBU Container [e07531f9e567 step-66ab04b33d] started, waiting for completion
[Sep 26 22:28:05] SERR ##### ERROR! MatPlotLib library could not be loaded!    #####
[Sep 26 22:28:05] SERR ##### Flat plots will instead be plotted as interactive #####
[Sep 26 22:28:05] SERR libstdc++.so.6: cannot open shared object file: No such file or directory
```

### PR DESCRIPTION
**Pull request by bgruening**
fix: https://github.com/conda-forge/matplotlib-feedstock/issues/61

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -56,6 +56,7 @@ requirements:
     - tk 8.5.*  # [linux]
     - functools32  # [py2k]
     - setuptools
+    - libgcc  # [linux]
```
Comment (jakirkham) [inline recipe/meta.yaml]: So we have had a lot of problems when adding `libgcc`. Namely some things get linked against system libraries when built and some things get linked to the conda supplied libraries. Hunting these issues down and resolving them was an endless source of headaches. Please see issue ( https://github.com/conda-forge/matplotlib-feedstock/issues/32 ) for the effects on `matplotlib`. However, these issues happened throughout our stack. Not inclined to go back down that road.

Comment (bgruening) [inline recipe/meta.yaml]: I had the impression that during one of the last meetings we decided to depend strictly on `gcc` or `llvm` exactly for those reasons.

Comment (jakirkham) [inline recipe/meta.yaml]: Not sure to what you are referring. For the most part, conda-forge tries to use system compilers and system libraries related to those compilers.

Comment (bgruening) [inline recipe/meta.yaml]: I think during one of the last meeting we agreed to explicitly depend on GCC and LLVM (OSX), isn't it? If we don't we still get https://github.com/conda-forge/matplotlib-feedstock/issues/61 on systems which does not have libgcc or gcc installed.

Comment (jakirkham) [inline recipe/meta.yaml]: That sounds right.

So we have been roughly following [manylinux]( https://www.python.org/dev/peps/pep-0513/#the-manylinux1-policy ) in terms of what we expect Linux to have. One of the things they expect there is `libstdc++`. So I think expecting `libstdc++` to be available on the VM or Docker image where `matplotlib` is installed is reasonable then. If we don't think this is reasonable, maybe we should revisit how we build things. Though this doesn't seem like a per feedstock discussion and seems more like an org level discussion.

**Comment by ocefpaf** [comment]
@bgruening what is the status on this one?

**Comment by bgruening** [comment]
I'm not sure. It seems I disagree with @jakirkham about what we should expect from a Linux distribution. I would like to run conda-packages in a busybox image and there is no libgcc installed. I also think that conda should define all dependencies and not assume some of them present.

As a practical consequences all packages that depends on matplotlib from conda-force do fail because of the missing libgcc library for us.

**Comment by ocefpaf** [comment]
@bgruening I agree with you but I lost that battle a long time ago and those with more free time got what they wanted without the proper discussion.

Maybe you could start a CFEP with those ideas and see if we can implement some discussion on the topic.

**Comment by bgruening** [comment]
> @bgruening I agree with you but I lost that battle a long time ago and those with more free time got what they wanted without the proper discussion.

That sounds bad. Maybe we should trigger a new discussion - what is a CFEP?

**Comment by ocefpaf** [comment]
> Maybe we should trigger a new discussion - what is a CFEP?

Conda-forge enhancement propoasal. See https://github.com/conda-forge/conda-forge-enhancement-proposals

The goal is to trigger a discussion.

**Comment by bgruening** [comment]
ping @johanneskoester

**Comment by johanneskoester** [comment]
I support this but it sounds like we would need quite some time to convince people. @bgruening what would be a suitable short term fix? Automatically add libgcc to the BusyBox containers from mulled?

**Comment by ocefpaf** [comment]
@bgruening I guess that only when we move to `conda-build 3` and start using the `ci3` compilers this will be finally solved. Closing for now but this issue is on our sights.

**Pull request closed without merging**